### DISCUSSION TITLE
Export space to PDF is initially limited to 100 pages; adding pages eventually errors out

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#572] [5. Export]
When I try to export my entire space, the total number of pages comes to 351. However, GitBook starts by only exporting 100 pages, and then displays an option to add an additional 50 pages. I can continue to incrementally add 50 pages, but it always errors out before I have the full space exported.

I realize that that is a large number of pages, but there aren't that many images and I'm surprised that it errors out completely.

Also surprised that this limitation (assuming it's a GitBook limitation and not a browser issue) doesn't seem to be documented.

**Comment by aikoii** [comment]
👋 @JaniceManwiller, it seems that in most cases (also reported in #609), refreshing the browser can resolve these PDF export issues. I realize this isn't perfect, but I've tested it with your docs, and it has worked. 🤞
